### ISSUE TITLE
getElementwiseStats in R 4.3.1

### ISSUE THREAD

**Issue by ericmalekos** [de-Boer-Lab/MAUDE#8]
I'm being met with `Error in xtfrm.data.frame(x) : cannot xtfrm data frames` when I call `getElementwiseStats` in R 4.3.1.

When I was using MAUDE on an earlier version of R, I did not have this error and searching the internet makes it seem that this error is being raised for a number of packages in R releases > 4, so that's my best guess...

Error:
```
elementLevelStatsUpper = getElementwiseStats(experiments = unique(binReadMat["expt"]),
                                             normNBSummaries = guideLevelStats,
                                             negativeControl = "isNontargeting",
                                             elementIDs=c("sgRNA", "Gene"),tails="upper")
Building background with 1650 non-targeting guides
Error in xtfrm.data.frame(x) : cannot xtfrm data frames
```

Variables:
```
> head(binReadMat, 3)
   Gene     sgRNA isNontargeting expt  hi  lo Background
1 Ccl12  Ccl12_g1          FALSE    A 539 251        395
2 Ccl12  Ccl12_g1          FALSE    B 519 464        491
3 Ccl12 Ccl12_g10          FALSE    A 925 468        696

> head(guideLevelStats, 3)
  expt               Gene            sgRNA isNontargeting   hi  lo Background  libFraction        mean         ll   llRatio           Z
1    A              Ccl12         Ccl12_g1          FALSE  747 381        405 2.369510e-05  0.09650030 -14.496249 12.291922  0.11107155
2    A ENSMUSG00000086424 86185803.23-P1P2          FALSE  773 747        601 3.516236e-05 -0.15124667 -13.095940 26.636367 -0.13667542
3    A              Ccl12        Ccl12_g10          FALSE 1133 598        706 4.130553e-05  0.05846178  -8.627353  8.802283  0.07303303

```

**Comment by ericmalekos** [comment]
Update: when I create a new R 4.0.5 environment in conda everything works again.

```
> elementLevelStatsUpper = getElementwiseStats(experiments = unique(binReadMat["expt"]),
                                             normNBSummaries = guideLevelStats,
                                             negativeControl = "isNontargeting",
                                             elementIDs=c("sgRNA", "Gene"),tails="upper")
elementLevelStatsUpper[elementLevelStatsUpper$FDR <= 0.1,]
Building background with 1650 non-targeting guides
```

**Comment by Carldeboer** [comment]
Thanks for posting this! This was caused by an update to the way the `order()` base function works. Previously, it could take as input a data.frame without reporting an error and we were inadvertently sending it single-column data.frame's rather than the array of data within the column itself in two places in the code. This was fixed in the release 1.0.2.
